src/main/java/ru/practicum/shareit/exception/ExceptionsHandler.java

```diff
@@ -0,0 +1,54 @@
+package ru.practicum.shareit.exception;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.method.annotation.MethodArgumentTypeMismatchException;
+
+@Slf4j
+@RestControllerAdvice
+public class ExceptionsHandler {
+
+    @ExceptionHandler
+    public ResponseEntity<ErrorResponse> constraintException(final MethodArgumentTypeMismatchException e) {
+        log.error("MethodArgumentTypeMismatchException ");
+        StringBuilder error = new StringBuilder();
+        error.append("Unknown ");
+        error.append(e.getName());
+        error.append(": ");
+        error.append(e.getValue());
+        return ResponseEntity.status(400).body(new ErrorResponse(error.toString()));
```

Comment:
Можно написать в одну строку
```suggestion
        return ResponseEntity.status(400).body(new ErrorResponse(
            new StringBuilder()
                .append("Unknown ")
                .append(e.getName())
                .append(": ")
                .append(e.getValue())
                .toString()));
```